### ISSUE TITLE
[Feature request] Add single window/instance mode

### ISSUE THREAD

**Issue by tdamsma** [ForkIssues/TrackerWin#1092]
As fork has integrated tabs I don't need multiple windows to be open. But as I usually have many applications open, I tend to use `Win+<first letters of app name>+Enter` to easily switch applications. In the case of Fork, this opens a new window each time. 

Could an option be added that enforces only a single window is open, and that if fork is opened again it just brings the already open window to the front?

**Comment by DanPristupov** [comment]
Win+number opens the existing Fork instance for me.

> Could an option be added that enforces only a single window is open, and that if fork is opened again it just brings the already open window to the front?

This is a duplicate of https://github.com/fork-dev/TrackerWin/issues/988 I guess

**Comment by tdamsma** [comment]
Yes it's a duplicate. Somehow I didn't notice that one, even though I tried searching, sorry. The Win+number trick only works if you know the order of fork on the taskbar, but I guess with pinning that can be solved. Will have at #266, some useful tips there

**Comment by Otiel** [comment]
@tdamsma In case you find it useful: I use [7+ Taskbar Numberer](https://rammichael.com/7-taskbar-numberer) in order to quickly know which <kbd>Win</kbd><kbd>Number</kbd> shortcut to hit:

![image](https://user-images.githubusercontent.com/3851902/106863128-5d50c980-66c8-11eb-951e-54314175085e.png)
